odoo/addons/test_main_flows/static/tests/tours/main_flow.js

```diff
@@ -887,7 +887,7 @@ tour.stepUtils.mobileModifier(tour.stepUtils.autoExpandMoreButtons('.o_control_p
     content: _t('See Project'),
     position: 'right',
 },
-tour.stepUtils.mobileModifier(tour.stepUtils.autoExpandMoreButtons('.o_control_panel .breadcrumb:contains("the_flow.project")')),
+tour.stepUtils.autoExpandMoreButtons('.o_control_panel .breadcrumb:contains("the_flow.project")'),
```

Comment:
Why do you changed this? It's just a question ^^

Reply:
yes i know it's a question, don't worry i really understand what is a code review 

the test was failing, i debugged and didn't find any error but finally i found that the button has become cached and that we expand more buttons to find it only on MobileModifier, then i think it should be done in web also

what do you think ?

Reply:
I don't really understand your explanation. I thought it was because we also have the expand more when we see this view in desktop and no longer only in mobile view.

Reply:
yes that's the same, i just explained why it work before my changes, because one button was not hidden and now it was hidden and we need to click on expand to see it